lib/matplotlib/axes/_axes.py

```diff
@@ -4103,113 +4094,137 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
-        :meth:`~matplotlib.collections.PolyCollection.get_array` on
-        this :class:`~matplotlib.collections.PolyCollection` to get
-        the counts in each hexagon. If *marginals* is *True*, horizontal
-        bar and vertical bar (both PolyCollections) will be attached
-        to the return collection as attributes *hbar* and *vbar*.
-
-
-        **Example:**
-
+        arguments.
+
+        cmap : object, optional
+            a :class:`matplotlib.colors.Colormap` instance. If *None*,
+            defaults to rc ``image.cmap``.
+           
+        norm : object, optional
+            :class:`matplotlib.colors.Normalize` instance is used to
+            scale luminance data to 0,1.
+
+        vmin : scalar, optional
+            *vmin* is used in conjunction with *norm* to normalize 
+            luminance data. If *None*, the min of the color array *C*
+            is used.  Note if you pass a norm instance, your settings
+            for *vmin* will be ignored.
+           
+        vmax : scalar, optional
+            *vmax* is used in conjunction with *norm* to normalize
+            luminance data. If *None*, the max of the color array *C*
+            is used.  Note if you pass a norm instance, your settings
+            for *vmax* will be ignored.
+
+        alpha : scalar between 0 and 1, optional
+            the alpha value for the patches
+
+        linewidths : scalar, optional
+            If *None*, defaults to 1.0. Note that this is a tuple, and
+            if you set the linewidths argument you must set it as a
+            sequence of floats, as required by
+            :class:`~matplotlib.collections.RegularPolyCollection`.
+
+        Other keyword arguments controlling the Collection properties.
+
+        edgecolors : {'none'}, mpl color, color sequence, optional
+            If 'none', draws the edges in the same color as the fill color.
+            This is the default, as it avoids unsightly unpainted pixels
+            between the hexagons.
+
+            If *None*, draws the outlines in the default color.
+
+            If a matplotlib color arg or sequence of rgba tuples, draws the
+            outlines in the specified color.
+           
+        Returns
+        -------
+        object
+            a :class:`~matplotlib.collections.PolyCollection` instance; use
+            :meth:`~matplotlib.collections.PolyCollection.get_array` on
+            this :class:`~matplotlib.collections.PolyCollection` to get
+            the counts in each hexagon.
+            
+            If *marginals* is *True*, horizontal
+            bar and vertical bar (both PolyCollections) will be attached
+            to the return collection as attributes *hbar* and *vbar*.
+        
+        Examples
+        --------
         .. plot:: mpl_examples/pylab_examples/hexbin_demo.py
+        
+        See also
+        --------
+        The standard descriptions of all the
```

Comment:
The "See Also" section is meant for other functions, methods or modules. This format will not be understood properly by numpdoc.
I think you should be using the "Notes" section instead of the "See Also" section.
It is also possible that the `%(Collection)s` needs tweaking, but I need to build the documentation to check this.